Enhancement for Issue 3: Adding ARM build support

decidedlygray opened the pull request:
Issue #3 - Request to merge changes to allow for building for ARM

binarycanary committed:
Enhancement for Issue 3: Adding ARM build support

zcutlip commented on buildarm.sh:
```diff
@@ -0,0 +1,51 @@
+	echo "$1" >&2
+}
+
+if [ ! -z $(which arm-linux-gcc-5.4.0) ];
```
can we get rid of the hard-coded version? there should be a symlink for a non-versioned name of the binary like `arm-linux-gcc` or something

zcutlip commented:
sorry, I know this is a few years old. would you like it merged still?

decidedlygray commented:
Hey I totally didn't see your comments before. I saw them by chance today while checking old issues. Yes I would like to merge still, I need to get rid of the hardcoded version. It's been a while since I've been back to this

zcutlip committed:
eliminate hard-coded gcc version

look for arm-linux-gcc instead of arm-linux-gcc-5.4.0

This will often be a symlink to the more specific versioned file.

zcutlip commented:
I made the edit to the arm shell script and merged. should be good to go
